packages/state/recoil/selectors/nft.ts

```diff
@@ -76,6 +79,35 @@ export const nftUriDataSelector = selectorFamily<
   },
 })
 
+export const allNftUsdValueSelector = selectorFamily<
+  number,
+  WithChainId<{ address: string }>
+>({
+  key: 'commonNftAllNftUsdValue',
+  get:
+    ({ chainId, address }) =>
+    ({ get }) => {
+      const accounts = get(accountsSelector({ chainId, address }))
+      const sum = get(
+        waitForAll(
+          accounts
+            .filter(
+              (x) =>
+                x.chainId === ChainId.StargazeMainnet ||
+                x.chainId === ChainId.StargazeTestnet
+            )
+            .map((x) =>
+              stargazeWalletUsdValueSelector({
+                chainId: x.chainId,
+                address: x.address,
+              })
```

Comment:
```suggestion
            .filter(
              ({ chainId }) =>
                chainId === ChainId.StargazeMainnet ||
                chainId === ChainId.StargazeTestnet
            )
            .map(({ chainId, address }) =>
              stargazeWalletUsdValueSelector({
                chainId,
                address,
              })
```

style nit: i prefer immediately destructuring fields like this, which makes the selector params object cleaner and gets rid of a meaningless variable (`x`). not a big deal and i don't expect you to perfectly follow my style preferences, but i wanted to let you know that's an option :)